azure_functions_worker/dispatcher.py

```diff
@@ -356,9 +358,19 @@ async def _handle__invocation_request(self, req):
                     trigger_metadata=trigger_metadata,
                     pytype=pb_type_info.pytype)
 
+            context = bindings.Context(
+                fi.name, fi.directory, invocation_id, trace_context)
+
+            # Execute before invocation callbacks
+            for callback in get_before_invocation_request_callbacks():
+                try:
+                    callback(context)
+                except Exception as ex:
+                    logger.warning(
+                        "Before invocation callback failed with: %s.", ex)
```

Comment:
@lzchen @Hazhzeng - How is this exposed to the user?

Reply:
Specifically - is this a USER log or System log? I think its system and won't be shown to the user. @Hazhzeng could you please confirm.

Reply:
This logger is a system log. Yes, it won't be shown to user.